src/vmm/src/persist.rs

```diff
@@ -414,6 +434,9 @@ mod tests {
     use snapshot::Persist;
     use utils::{errno, tempfile::TempFile};
 
+    #[cfg(target_arch = "aarch64")]
+    const FC_VERSION_0_23_0: &str = "0.23.0";
+
     fn default_vmm_with_devices(event_manager: &mut EventManager) -> Vmm {
```

Comment:
```suggestion
    fn default_vmm_with_devices() -> Vmm {
        let mut event_manager = EventManager::new().expect("Cannot create EventManager");
```
I know this is out of the scope of this PR but it would be nice to have a function that just gives you a default_vmm so that we can use it to test "test_get_snapshot_data_version". WDYT?

Reply:
Good idea.